connectors/src/connectors/google_drive/temporal/file.ts

```diff
@@ -470,11 +473,12 @@ async function upsertGdriveDocument(
   const documentLen = documentContent ? sectionLength(documentContent) : 0;
 
   if (documentLen > 0 && documentLen <= maxDocumentLen) {
-    const parents = (
-      await getFileParentsMemoized(connectorId, oauth2client, file, startSyncTs)
-    ).map((f) => f.id);
-    parents.push(file.id);
-    parents.reverse();
+    const parents = await getFileParentsForUpsert(
```

Comment:
:+1: did you check all other calls to getFileParentsMemoized?

Reply:
> did you edit my message 😮 😄 ?

🙈 🤣